pool/postgres.go

```diff
@@ -35,6 +35,15 @@ func InitPostgresDB(host string, port uint32, user, pass, dbName string) (*Postg
 		return nil, errs.DBError(errs.DBOpen, desc)
 	}
 
+	pdb := &PostgresDB{db}
+	if purgeDB {
+		err := pdb.purge()
+		if err != nil {
+			desc := fmt.Sprintf("%s: unable to purge db: %v", funcName, err)
```

Comment:
`purge()` is already wrapping the error with `errs.DeleteEntry`. Don't need to wrap it again here.